soc/tests/test_BatteryPower.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import os.path
+import pytest
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from BatteryPower import BatteryPower
+
+@pytest.fixture
+def battery():
+    battery = BatteryPower()
+    return battery
+
+#testing using sample data under the title BATTERY_AGGREGATE_VC from can_messages.csv
+def test_battery_aggregate_vc(battery):
```

Comment:
Note for future: might be more interesting to read through `can_messages.csv` directly. Once some more things get implemented, test could be create battery pack model instance -> read through csv -> `calc_power` -> discharge -> assert. I doubt it's the finest of unit testing practices, but might be more helpful.